### ISSUE TITLE
Real time output of pre commit/push hooks

### ISSUE THREAD

**Issue by Dhq** [kahole/vscode-magit#103]
First of all, huge thank you for a fantastic port of the awesome magit. Great job :) 

Pretty much the only thing that bothers me is that I find it hard to follow on the progress of my pre commit/push hooks in the process.magit window. Say I commit, the last line in that window will be `git commit` until all the tests/linting in my hooks complete and I get shown the commit dialog.

Am I missing something? If not, it could be considered a feature to implement 👍

**Comment by kahole** [comment]
So it's all there at the end, but you would like it to stream the output to the process log instead?
It is something to consider :+1:

**Comment by max-mykhailenko** [comment]
Hi! I think my case is very similar. So after commit message validation via git hook I want to show feedback for developer. And only one place where I found it — output git console. Is it possible to show some alert instead?

And yes, plugin is awesome!

**Comment by max-mykhailenko** [comment]
@kahole sorry for ping, but really interesting if possible to do something with that, I can contribute if you give quick workaround about project structure and place where I should check it

**Comment by kahole** [comment]
Does it show in the edamagit process log (press '$' to open it)

If it does, one solution could be to have a setting-toggle for auto-opening it on commit.

**Comment by max-mykhailenko** [comment]
> Does it show in the edamagit process log (press '$' to open it)
> 
> If it does, one solution could be to have a setting-toggle for auto-opening it on commit.

I see logs in `process.magit`  

```
[0] git add -u -- /Users/max/Projects/frontend-template/Readme.md
[1] git commit
Bad commit message, see example: TASK-123 commit message 
 Current message: null test commit
# Please enter the commit message for your changes. Lines starting
# with '#' will be ignored, and an empty message aborts the commit.
#
# On branch master
# Your branch is up to date with 'origin/master'.
#
# Changes to be committed:
#	modified:   Readme.md
#

husky - commit-msg hook exited with code 1 (error)
```

**Comment by kahole** [comment]
I think I have a solution that might work for you ^
Will be part of the next release, today or tommorow

**Comment by kahole** [comment]
v0.6.15 released now

**Comment by max-mykhailenko** [comment]
@kahole Thank you, for the fast solution, it's awesome!

<img width="859" alt="image" src="https://user-images.githubusercontent.com/1213198/121785706-0605bd00-cbc4-11eb-9d1d-1784c1914722.png">
